haas/cli.py

```diff
@@ -123,6 +123,10 @@ def request(self, method, url, data=None, params=None):
 http_client = None
 
 
+class InvalidAPIArgumentsException(Exception):
```

Comment:
If that one should, so should the other exception there.  I'd rather keep client-side errors separate from server-side errors.